### ISSUE TITLE
Error on `Phantom-Types`

### ISSUE THREAD

**Issue by LiamGoodacre** [JordanMartinez/purescript-jordans-reference#45]
This document presents the following code:

```purescript
newtype Attribute = Attribute { key :: String, value :: String }

attribute :: String -> String -> Attribute
attribute = Attribute { key: _, value: _ }
```

The compiler will throw a type error for the `attribute` function.
The `{ key: _, value: _ }` notation is syntax for `\v0 v1 -> { key: v0, value: v1 }`.
So `Attribute { key: _, value: _}` is actually `Attribute (\v0 v1 -> { key: v0, value: v1 })`.

To make this work I'd suggest:

```purescript
attribute :: String -> String -> Attribute
attribute key value = Attribute { key, value }
```

The overly confusing point-free version is:
```purescript
attribute' :: String -> String -> Attribute
attribute' = (Attribute <<< _) <<< { key: _, value: _ }
```

**Comment by JordanMartinez** [comment]
Yeah... I haven't really done much work on the Design Pattern folder since initially creating it and when I was first learning Purescript. I should probably explain that it's still a WIP and redirect people to issues tagged with the Design Patterns label

Thanks for correcting that though.

**Comment by JordanMartinez** [comment]
Closed by #95
